devutils/src/devutils/invoke_utils.py

```diff
@@ -390,30 +445,63 @@ def make_release(ctx):
             echo=True,
         )
 
-        print(f"Trigger the release with: git push origin {current_tag}")
+        print(f"Pushing tag: {current_tag}")
+        run(ctx, f"git push origin {current_tag}")
 
     @task
     def check_tag_version(ctx):
         """
         Checks if the current tag matches the latest version (exits with 1 if it
         does not match and with 0 if it does match).
         """
-        import importlib
-
-        mod = importlib.import_module(package_name)
+        module_version = poetry(
+            ctx,
+            f"run python -c 'import {package_name}; print({package_name}.__version__)'",
+        ).stdout.strip()
 
         tag = get_tag(tag_prefix)
         version = tag[tag.rfind("-") + 1 :]
 
-        if mod.__version__ == version:
+        if module_version == version:
             sys.stderr.write(f"Version matches ({version}) (exit(0))\n")
             sys.exit(0)
         else:
             sys.stderr.write(
-                f"Version does not match ({tag_prefix}: {mod.__version__} != repo tag: {version}).\nTags:{get_all_tags(tag_prefix)}\n(exit(1))\n"
+                f"Version does not match ({tag_prefix}: {module_version} != repo tag: {version}).\nTags:{get_all_tags(tag_prefix)}\n(exit(1))\n"
             )
             sys.exit(1)
 
+    def update_changelog_file(file: Path, version: str):
+        """Update the changelog file with the new version and changes"""
+
+        with open(file, "r+") as stream:
+            content = stream.read()
+
+            # Prepend a new section with the new version and changes
+            new_version_section = (
+                f"## {version} - {datetime.today().strftime('%Y-%m-%d')}\n\n"
+            )
+
+            changelog_start = re.search(r"# Changelog", content).end()
+            last_version_pos = re.search(r"## (\d+\.\d+\.\d+)", content).start()
+
+            double_newline = "\n\n"
+            unreleased_changes = content[changelog_start:last_version_pos].strip()
+
+            if unreleased_changes:
+                new_version_section += unreleased_changes + double_newline
+
+            new_content = (
+                content[:changelog_start]
+                + double_newline
+                + new_version_section
+                + content[last_version_pos:]
+            )
+
+            stream.seek(0)
+            stream.write(new_content)
+            print(print("Changed: ", file))
+
     @task
     def set_version(ctx, version):
```

Comment:
```suggestion
    def set_version(ctx, version: str):
```

Would be awesome if you can parse the version and check if it's numeric, as I'm always accidentally running: `inv set-version -h` to see some usage and guess what, I end up with this version:
```
__version__ = "-h"
```

You can use this [pattern](https://github.com/robocorp/rpaframework/blob/master/packages/core/src/RPA/core/certificates.py#L45): `packaging.version.parse()` to parse and check the atoms if are integers. (ar at least 3 in number, as we're following semver)